src/main/target/FRSKYF3/target.h

```diff
@@ -21,6 +21,17 @@
 #pragma once
 
 #define TARGET_BOARD_IDENTIFIER "FRF3"
+
+// prevent flash overflow
+#undef USE_CRSF_CMS_TELEMETRY
```

Comment:
Do all of these have to be removed to make it build? There is a list in `target.h` for OMNIBUS that is listing candidates for removal, in order of priority that should be used across all targets.